components/drafts/drafts_link.tsx

```diff
@@ -22,18 +22,17 @@ const getDraftsCount = makeGetDraftsCount();
 function DraftsLink() {
     const dispatch = useDispatch();
     const localDraftsEnabled = useSelector(localDraftsAreEnabled);
-    const syncDraftsEnabled = useSelector(syncedDraftsAreAllowedAndEnabled);
     const {formatMessage} = useIntl();
     const {url} = useRouteMatch();
     const match = useRouteMatch('/:team/drafts');
     const count = useSelector(getDraftsCount);
     const teamId = useSelector(getCurrentTeamId);
 
     useEffect(() => {
-        if (syncDraftsEnabled) {
+        if (localDraftsEnabled) {
             dispatch(getDrafts(teamId));
         }
-    }, [teamId, syncDraftsEnabled, dispatch]);
+    }, [teamId, dispatch, localDraftsEnabled]);
```

Comment:
That makes sense to me as long as we don't need to refetch if the old `syncDraftsEnabled` changes

Reply:
Do we need though to reconcile since we won't have any server drafts?
Local drafts already exist as "Global Items", and will be already migrated, so we don't need to save them again.

Reply:
Yeah you're right, this can be reverted back to its original form, and `getDrafts` can be reverted as well.

Reply:
So to resolve the draft reconciliation issue, I've moved the `syncDraftsAllowedAndEnabled` check back to `drafts_link.tsx`, and updated `getDrafts` to just get the local drafts from storage and merge them with server drafts.

I've had to modify `makeGetDrafts` as well to take in a boolean `excludeInactive` that changes how we filter drafts. On page load the `getMyActiveChannelIds` selector doesn't return anything and so the default `makeGetDrafts` wouldn't return any drafts. I tried to find a way to use the existing selector while also avoiding any unnecessary network requests or refreshes, but because we're updating the storage in `getDrafts` it wasn't really possible. This won't affect how we display drafts as there are no network requests being made there so we can wait for `getMyActiveChannelIds` to filter the drafts accordingly.